vector/v.lrs/lrs.html

```diff
@@ -167,14 +167,16 @@ <h2>NOTES</h2>
 
 <h2>SEE ALSO</h2>
 
-<em>R. Blazek, 2004, <a href="http://gisws.media.osaka-cu.ac.jp/grass04/viewpaper.php?id=50">Introducing the Linear Reference System in GRASS</a>, Bangkok, GRASS User Conf. Proc.</em><br>
-<em>R. Blazek, 2005, <a href="http://creativecity.gscc.osaka-cu.ac.jp/IJG/article/download/320/321">Introducing the Linear Reference System in GRASS</a>, International Journal of Geoinformatics, Vol. 1(3), pp. 95-100</em><br>
+<em>R. Blazek, 2004, <a href="https://web.archive.org/web/20070301130303/http://gisws.media.osaka-cu.ac.jp/grass04/viewpaper.php?id=50">Introducing the Linear Reference System in GRASS</a>, Bangkok, GRASS User Conf. Proc.</em><br>
```

Comment:
Maybe https://www.geoinfo-lab.org/grass04/Full%20Paper_PDF/Introducing%20the%20Linear%20Reference%20System%20in%20GRASS.pdf?
Found here : https://www.geoinfo-lab.org/grass04/Papers.html
from https://www.geoinfo-lab.org/grass04/index.html

The website of the conference seems hosted at https://github.com/Geoinformatics-Lab/geoinformatics-lab.github.io/tree/3ac0520c5f205d04b6052cd30b489973a7e1df5d/grass04

Reply:
Perfect, PDF URL updated accordingly.

Reply:
I found a better reference:
https://foss4g.asia/2004/
-> PDF: https://foss4g.asia/2004/Full%20Paper_PDF/Introducing%20the%20Linear%20Reference%20System%20in%20GRASS.pdf